packages/api/endpoints/async-operations.js

```diff
@@ -29,17 +44,15 @@ async function list(req, res) {
  * @returns {Promise<Object>} the promise of express response object
  */
 async function getAsyncOperation(req, res) {
-  const asyncOperationModel = new AsyncOperationModel({
-    stackName: process.env.stackName,
-    systemBucket: process.env.system_bucket,
-    tableName: process.env.AsyncOperationsTable,
-  });
+  const knex = await getKnexClient({ env: process.env });
+  const asyncOperationPgModel = new AsyncOperationPgModel();
 
   let asyncOperation;
   try {
-    asyncOperation = await asyncOperationModel.get({ id: req.params.id });
+    const asyncOperationRecord = await asyncOperationPgModel.get(knex, { id: req.params.id });
+    asyncOperation = translatePostgresAsyncOperationToApiAsyncOperation(asyncOperationRecord);
```

Comment:
Something like this? [`794ba58` (#2294)](https://github.com/nasa/cumulus/pull/2294/commits/794ba5846ab5af090a7a61da2d60d2e45ab6cef6)

This doesn't entirely get rid of `let`, since we need another one